gravityflow/gravityflowformconnector
# Fix PHP 7.2+ issue

re: #8 

Fixes the `create_function` deprecation notice with PHP 7.2.

**Testing Instructions**

- Create a form with a few single input and multi-input fields.
- Duplicate the form.
- Add a form submission type step to the first form. Select the duplicate form as the form to be used by the step and map the fields.
- Submit the first form.
- Navigate to the inbox, view the entry and click the open form button.

With master and PHP 7.2:  See the PHP notice and the values have been populated.
With this branch and PHP 7.2 find the notice no longer occurs and the values are still populated. Check older PHP versions and find the values are still populated.
